terraform/main.tf

```diff
@@ -4,6 +4,13 @@ module "quorum_cluster" {
 
   # Variables sourced from terraform.tfvars
   public_key_path                    = "${var.public_key_path}"
+  private_key_path                   = "${var.private_key_path}"
+  backup_interval                    = "${var.backup_interval}"
+  BackupLambda_source_file           = "${var.BackupLambda_source_file}"
+  BackupLambda_output_path           = "${var.BackupLambda_output_path}"
```

Comment:
nit: Let's use `backup_lambda` rather than `BackupLambda` for these. I understand you were shooting for consistency with the name of the application but I'd rather not introduce camel case into the Terraform configs.

Reply:
These are now using backup_lambda...